notes/roman-urdu/77_Surah_Mursalat.md

```diff
@@ -0,0 +1,26 @@
+# Number 77: Surah Mursalat  
+
+Classification: Makki  
+Rukus: 2  
+Ayahs: 50  
+Subject: Qayyamat  
+Related Surah: Surah Naba  
+
+Qayyamat ka naqsha.  
+- Hawaein/farishton ki qasam; k woh waqia (qayyamat) ho kr rhe ga.
+- Faislay ka din (halakat hai us din jutlany walo k liye) yeh ayat 8 baar repeat hoti h.
```

Comment:
I would recommend the following convention:

1. Use the 3 hashes for defining a block's heading i.e. ```### Heading```
2. Use numbers list instead of unordered list as that is the format current notes are following

For example:

### Qayyamat ka naqsha
1. Hawaein/farishton ki qasam; k woh waqia (qayyamat) ho kr rhe ga.
2. Faislay ka din (halakat hai us din jutlany walo k liye) yeh ayat 8 baar repeat hoti h.